packages/react-aria-components/docs/Toolbar.mdx

```diff
@@ -38,27 +38,20 @@ type: component
 ## Example
 
 ```tsx example
-import {Toolbar, Button, ToggleButton, Separator, Popover, MenuTrigger, Menu, MenuItem, Checkbox} from 'react-aria-components';
-
-<Toolbar>
-  <ToggleButton><strong>B</strong></ToggleButton>
-  <ToggleButton><div style={{textDecoration: 'underline'}}>U</div></ToggleButton>
-  <ToggleButton><i>I</i></ToggleButton>
+import {Toolbar, Button, ToggleButton, Separator, Checkbox, Group} from 'react-aria-components';
+
+<Toolbar aria-label="Text formatting">
+  <Group aria-label="Style">
+    <ToggleButton aria-label="Bold"><b>B</b></ToggleButton>
+    <ToggleButton aria-label="Italic"><i>I</i></ToggleButton>
+    <ToggleButton aria-label="Underline"><u>U</u></ToggleButton>
+  </Group>
   <Separator orientation="vertical" />
-  <Button>Copy</Button>
-  <Button>Paste</Button>
-  <Button>Cut</Button>
-  <Separator orientation="vertical" />
-  <MenuTrigger>
-    <Button>Alignment</Button>
-    <Popover>
-      <Menu>
-        <MenuItem>Left</MenuItem>
-        <MenuItem>Center</MenuItem>
-        <MenuItem>Right</MenuItem>
-      </Menu>
-    </Popover>
-  </MenuTrigger>
+  <Group aria-label="Clipboard">
+    <Button>Copy</Button>
+    <Button>Paste</Button>
+    <Button>Cut</Button>
+  </Group>
```

Comment:
Added missing aria-labels and groups. Removed the menu because the example was already pretty long, and it seemed like this particular example should really be a Select (so you could actually select an alignment). I tried a Select but it doesn't seem to work at the moment.

Reply:
yeah, that's why it's not listed in the group of supported components as children right now. Until we can decide how to handle this (we tried in some earlier commits https://github.com/adobe/react-spectrum/pull/5080/commits/8e7940e14424e18d70c9e4f87bdc82746f4c737d#diff-83790784ea28a9ab299b48d17c2a9652f39f3a10b34a8745201869d3a1bdb6ffL57), I have left it out and placed a menu in because that can achieve roughly the same thing

Reply:
these Groups are nice, i'll have to add them to our storybook too